src/monodroid/monodroid.targets

```diff
@@ -22,22 +22,41 @@
         Overwrite="True"
     />
     <Exec
-        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) V=1"
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) NDK_LIBS_OUT=./debug/libs NDK_OUT=./debug/obj V=1"
```

Comment:
That's simple...
![screenshot-encrypted google com 2016-07-07 00-17-55](https://cloud.githubusercontent.com/assets/53929/16624768/fb347318-43dc-11e6-8a65-afd719d604b7.png)